components/Modal/CreateImage.js

```diff
@@ -128,12 +129,16 @@ class CreateImage extends React.Component {
                   ><FormattedMessage defaultMessage="Blueprint" /></label>
                   <div className="col-sm-9">
                     <p className="form-control-static">{this.props.blueprint.name}</p>
+                    	<label
+                    		data-id="cmpsr-form-ctrl"
+                    	>
                   </div>
                 </div>
                 <div className="form-group">
                   <label
                     className="col-sm-3 control-label"
                     htmlFor="textInput-modal-markup"
+                    data-id="cmpsr-modal-markup1"
```

Comment:
Instead of adding a `data-id` attribute here, could you add an `id` to the `<select>` as noted in a [previous comment](https://github.com/weldr/welder-web/pull/375/files#diff-886584fb195b433a216cecc7e230cd8cR27)